src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/TextureManager.cs

```diff
@@ -292,10 +422,12 @@ private void SetupCameraAndRenderTextures()
             spectatorViewCamera.targetTexture = renderTexture;
 
             colorRGBTexture = new RenderTexture(frameWidth, frameHeight, (int)Compositor.TextureDepth);
+            videoFeedColorCorrectionTexture = new RenderTexture(frameWidth, frameHeight, (int)Compositor.TextureDepth);
             alphaTexture = new RenderTexture(frameWidth, frameHeight, (int)Compositor.TextureDepth);
             compositeTexture = new RenderTexture(frameWidth, frameHeight, (int)Compositor.TextureDepth);
             occlusionMaskTexture = new RenderTexture(frameWidth, frameHeight, (int)Compositor.TextureDepth);
             blurOcclusionTexture = new RenderTexture(frameWidth, frameHeight, (int)Compositor.TextureDepth);
+            blurOcclusionHelperTexture = new RenderTexture(frameWidth, frameHeight, (int)Compositor.TextureDepth);

```

Comment:
>blurOcclusionHelperTexture = new RenderTexture(frameWidth, frameHeight, (int)Compositor.TextureDepth); [](start = 12, length = 102)

What is this for? I don't see it used here anywhere

Reply:
i had thought i would need this for the multi-pass blurring but it looks like it isn't needed.